src/state/genmoReducers.js

```diff
@@ -52,11 +86,15 @@ function followLinkReducer(state, action) {
             if (!data[key]) {
               if (!currentPassage.needsPrompt) currentPassage.needsPrompt = [];
               const keyIndex = currentPassage.needsPrompt.findIndex(
-                p => p.key === key
+                (p) => p.key === key
               );
 
               if (keyIndex < 0) currentPassage.needsPrompt.push({ key });
             }
+          } else if (key === SPECIAL_DATA_KEYS.INVENTORY_ADD) {
+            updateInventory(data, value, 1);
+          } else if (key === SPECIAL_DATA_KEYS.INVENTORY_REMOVE) {
+            updateInventory(data, value, -1);
           } else {
             data[key] = value;
```

Comment:
Should add some protection around the `inventory` key, so that it won't be overwritten if someone decides to use it in their story.